docs/user/dashboard/lens.asciidoc

```diff
@@ -69,23 +81,20 @@ Formulas allow you to perform math on aggregated data. The most common formulas
 
 . Click *Formula*, then enter the formula. 
 +
-Filter ratio example:: Use `kql=''` to filter one set of documents, then compare the document set to other documents within the same grouping:
-+
-See how the error rate changes over time:
+Filter ratio example:: To filter a document set, use `kql=''`, then compare to other documents within the same grouping:
 +
 ```
 count(kql='response.status_code > 400') / count()
 ```
 +
-Week over week example:: Use `shift='1w'` to get the value of each grouping from the previous week. Time shift should not be used with the *Top values* function.
+Week over week example:: To get the value for each grouping from the previous week, use `shift='1w'`. Do not use time shift formulas with the *Top values* function.
```

Comment:
How about: If your formula contains a time shift **and** a non-shifted metric then you cannot currently use Top Values due to a current limitation.

Reply:
I think the current error message says it better, as the problem is with multiple different `time shift` rather than time shift vs non-time shift:

> It is not possible to combine different time shifts with the *Top values* function.

Reply:
I think specifying the different time shifts instead of saying `different` is clearer. On the other hand, using `different` is a simpler way of stating this, especially when I finally understood the different time shifts "types".

Reply:
@dej611 is better at english than I am. my parents would be so proud 😂 

nice one, sounds much better.

Reply:
> I think specifying the different time shifts instead of saying `different` is clearer. On the other hand, using `different` is a simpler way of stating this, especially when I finally understood the different time shifts "types".

I thought about different formulations, but the main point, at the end, is not mixing them:

```
count(shift="1w") - count() // "different": in this case is 1w vs 0 shift
count(shift="1w") - count(shift="1m") // "different": in this case is 1w vs 1m
```

Of course when `Top values` is selected as function on other dimensions.
We might think to include this type of example as well to better explain, but it's probably too much.